modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/resource-actions/default.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<!DOCTYPE resource-action-mapping PUBLIC "-//Liferay//DTD Resource Action Mapping 7.4.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_4_0.dtd">
+
+<resource-action-mapping>
+	<portlet-resource>
+		<portlet-name>com_liferay_frontend_data_set_views_web_internal_portlet_FDSViewsPortlet</portlet-name>
+		<permissions>
+			<supports>
+				<action-key>MANAGE_FDS_VIEWS_ENTRIES</action-key>
```

Comment:
A quick look to the rest of `<portlet-resource>` entries reveals that, with very few exceptions, the actions declared are related to the portlet itself (kinda: add to page, add ADT, configure, etc). Declaring model-related actions in the portlet is not common at all.

This is not to say we should not do it. Indeed we might need to.

What's the purpose of `MANAGE_FDS_VIEWS_ENTRIES`? is it the equivalent to `create` for the resource type `FDSEntry`? or also includes the ability to delete and edit them?

I ask because Object permissions are administerable on their own:
![image](https://github.com/liferay-frontend/liferay-portal/assets/861014/b3f86e5c-7a74-49b3-bc11-c4cba06899c1)

There we have:

- Application permissions: sadly, not reusable for the dataset app because they regulate access to the auto-generated widget for that object definition in question
- Resource type permissions (FDS ENTRIES, note the plural): control access to the creation and administration of object entries of this type. So, there is an action dedicated to "Create Datasets" here that we can use to enforce access to the functionality
- Resource permissions (FDS ENTRY): control access to individual object entries of that type. Those should appear in the permission modal associated to each particular dataset entry

An additional note is that, for these permissions to be administerable as you see in the screenshots, one has to make the object definition "public" by giving it a "Panel Category Key", otherwise, the "FDS Entry" won't be found when defining role permissions.

I think we should be able to reuse this as much as we can for the "resource permissions" part, then define widget permissions at the dataset portlet level as any other portlet.